climada/util/config.py

```diff
@@ -50,6 +50,21 @@ class Config():
     Like this it's possible to refer to e.g. test files in the climada sources.
     """
 
+    def __getattribute__(self, __name):
+        # the only reason for overriding __getattribute__: better error messages in case a
+        # configuration value is asked for but never defined in any of the possible config files
+        try:
+            return super().__getattribute__(__name)
+        except AttributeError:
+            conf_files = [Path(_find_in_parents(conf_dir, CONFIG_NAME))
+                          if _find_in_parents(conf_dir, CONFIG_NAME)
+                          else conf_dir / CONFIG_NAME
+                          for conf_dir in CONFIG_DIRS[::-1]]
```

Comment:
I do not fully understand where the variables `CONFIG_NAME` and `CONFIG_DIRS` are defined (or imported).

Reply:
Because `CONFIG` kind of _has_ to be at the bottom of the file and both `CONFIG_NAME` and `CONFIG_DIRS` have _one_ purpose: to define `CONFIG`. So I want them to be together, then I don't have to wonder or scroll.